python/src/scipp/compat/mantid.py

```diff
@@ -52,6 +53,69 @@ def make_run(ws):
     return sc.Variable(value=deepcopy(ws.run()))
 
 
+additional_unit_mapping = {
+    "Kelvin": sc.units.K,
+    "microsecond": sc.units.us,
+    "nanosecond": sc.units.ns,
+    "second": sc.units.s,
+    "Angstrom": sc.units.angstrom,
+    "Hz": sc.units.dimensionless / sc.units.s,
+    "degree": sc.units.deg,
+}
+
+
+def make_variables_from_run_logs(ws):
+    lookup_units = dict(
+        zip([str(unit) for unit in sc.units.supported_units()],
+            sc.units.supported_units()))
+    lookup_units.update(additional_unit_mapping)
+    for property_name in ws.run().keys():
+        units_string = ws.run()[property_name].units
+        units = lookup_units.get(units_string, None)
```

Comment:
`unit` would be a clearer name, if this isreferingto a single unit?

Reply:
I guess you are right, the string usually states the "units", for example "kilograms", and this is mapped to the unit: "kg".